MANUAL.txt

```diff
@@ -976,8 +976,8 @@ Options affecting specific writers {.options}
     include multiple files. They will be included in the order specified.
 
     A stylesheet is required for generating EPUB.  If none is
-    provided using this option (or the `stylesheet` metadata
-    field), pandoc will look for a file `epub.css` in the
+    provided using this option (or the deprecated `stylesheet`
+    metadata field), pandoc will look for a file `epub.css` in the
```

Comment:
I think this is a bit misleading:  if you use `css` in the metadata, that's not the same as using `--css` command line option.  Better would be:  "provided using this option (or the `css` or `stylesheet` metadata field)"